Analysis/idsf.tex

```diff
@@ -95,6 +104,9 @@
   \caption{\egamma\ scale factors as a function of photon \pt.}
   \label{tab:idsf_results}
 \end{table}
+\PH{Maybe I misse this earlier, but I had to think for a sec why the
+  MC is not 1. It might be good to explicitly explain there are two
+  SFs e to $\gamma$ and Data to MC}
```

Comment:
I understand the point, but also it could just be from MC stats and measurement error. not necessarily the e->gamma SF. will address if you really want me to, but I think it just complicates things